scheduler/node/node.go

```diff
@@ -0,0 +1,86 @@
+package node
```

Comment:
Is this package really necessary? All it contains is a single struct.

Also, if it was called `scheduler.Node` it would perhaps be more clear that this is a scheduler thing (much like if it was called `scheduler.Candidate` or `scheduler.Destination`)